packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -56,6 +63,38 @@ import type {BrowserTheme} from 'react-devtools-shared/src/frontend/types';
 import type {ReactFunctionLocation, ReactCallSite} from 'shared/ReactTypes';
 import type {SourceSelection} from './Editor/EditorPane';
 
+const MaybeActivity: any =
```

Comment:
I don't know how to type this. Flow treats `React.Activity` as a Symbol not a ElementType with certain props.